ObjectCreation/BSpline3D.FCMacro

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+#***************************************************************************
+#                          BSpline3D
+#***************************************************************************
+# Create a parametric BSpline from Vertices selected in 3D view
+# Select two or more vertices in the 3D view, then execute the macro.
+#***************************************************************************
+#*   Copyright (c) 2024 Eric Price (CorvusCorax)                           *
+#*                      <eric.price[at]tuebingen.mpg.de>                   *
+#*                                                                         *
+#*   This file is an extension of FreeCAD.                                 *
+#*                                                                         *
+#*   FreeCAD is free software: you can redistribute it and/or modify it    *
+#*   under the terms of the GNU Lesser General Public License as           *
+#*   published by the Free Software Foundation, either version 2.1 of the  *
+#*   License, or (at your option) any later version.                       *
+#*                                                                         *
+#*   FreeCAD is distributed in the hope that it will be useful, but        *
+#*   WITHOUT ANY WARRANTY; without even the implied warranty of            *
+#*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU      *
+#*   Lesser General Public License for more details.                       *
+#*                                                                         *
+#*   You should have received a copy of the GNU Lesser General Public      *
+#*   License along with FreeCAD. If not, see                               *
+#*   <https://www.gnu.org/licenses/>.                                      *
+#*                                                                         *
+#**************************************************************************/
+
+_Name__ = 'BSpline3D'
+__Comment__ = 'Create a parametric BSpline from Vertices selected in 3D view'
+__Author__ = 'CorvusCorax'
+__Version__ = '0.0.1'
+__Date__ = '2024-11-11'
+__License__ = 'LGPL-2.1-or-later'
+__Web__ = 'https://github.com/CorvusCorax/FreeCAD-BSpline3D'
```

Comment:
You should then remove your repo if you want to avoid to have the code in multiple places.

Reply:
i wanted to use that to have a wiki with usage information - so two flies with one swat - i have a temp repo for updates/pull requests and i have a wiki for documentation